packs/github/actions/lib/base.py

```diff
@@ -16,11 +16,12 @@ def __init__(self, config):
         super(BaseGithubAction, self).__init__(config=config)
         token = self.config.get('token', None)
         token = token or None
-        self._client = Github(token)
+        base_url = self.config.get('base_url', None)
```

Comment:
yes and no. this allows the user to remove the base_url from the config and it'll still fall back to this default. just a last resort fallback. :)